tests/lib.rs

```diff
@@ -27,9 +27,12 @@ struct TestEnv {
 impl TestEnv {
     fn new() -> Self {
         TestEnv {
-            cache_dir: TempDir::new(".tldr.test.cache").unwrap(),
-            config_dir: TempDir::new(".tldr.test.config").unwrap(),
-            input_dir: TempDir::new(".tldr.test.input").unwrap(),
+            cache_dir: Builder::new().prefix(".tldr.test.cache").tempdir().unwrap(),
+            config_dir: Builder::new()
+                .prefix(".tldr.test.config")
+                .tempdir()
+                .unwrap(),
```

Comment:
If you want to avoid this re-wrapping, change the prefix to `.tldr.test.conf` :slightly_smiling_face: